frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -247,5 +247,6 @@ export const detectGitRepoName = (url: string): string | undefined => {
   return url
     .split('/')
     .pop()
-    .replace('.', '-');
+    .replace('.', '-')
+    .toLowerCase();
```

Comment:
Have you considered `_.kebabCase`? It should handle these special cases a likely others

https://lodash.com/docs/4.17.15#kebabCase

Reply:
I did not consider ```_.kebabCase```. I will look into it. Thanks.

Reply:
@spadgett Now using ```kebabCase``` instead of manually replacing the ```.```, ```_``` and convert to lower case.